src/deprecation.h

```diff
@@ -6,7 +6,7 @@
 #define ZCASH_DEPRECATION_H
 
 // Deprecation policy is 4th third-Tuesday after a release
```

Comment:
This is no longer correct. Does not block release, but let's fix this comment afterward.